project/SparkBuild.scala

```diff
@@ -733,7 +734,8 @@ object Unidoc {
     unidocSourceBase := s"https://github.com/apache/spark/tree/v${version.value}",
 
     scalacOptions in (ScalaUnidoc, unidoc) ++= Seq(
-      "-groups" // Group similar methods together based on the @group annotation.
+      "-groups", // Group similar methods together based on the @group annotation.
+      "-skip-packages", "org.apache.hadoop"
```

Comment:
For the scala, we can not exclude `org.apache.hadoop.hive.ql.io.orc` because `scalac` makes build errors about the missing type in `org.apache.hadoop.hive.ql.io.orc`. Instead, we need to use `-skip-packages` option.